gradio/components/base.py

```diff
@@ -190,9 +190,6 @@ def __init__(
         self.scale = scale
         self.min_width = min_width
         self.interactive = interactive
-        # Keep tracks of files that should not be deleted when the delete_cache parmaeter is set
```

Comment:
Please add `keep_in_cache=True` to `move_files_to_cache` below

Reply:
Ah yeah thanks